src/main.rs

```diff
@@ -151,12 +148,11 @@ fn main() {
 
             // TODO: create game instance + AI
             println!("--- Single-Player-Mode ---");
-            mode = Mode::Single;
+            model::start_round();
```

Comment:
The name is not used anymore. Wouldn't it be nice, if the name of the single-player would be used instead of output like "Player 1, please enter...". So my suggestion would be: `start_round(name)`